conans/client/installer.py

```diff
@@ -207,11 +207,17 @@ def install(self, deps_graph, remotes):
         install_graph.raise_errors()
         install_order = install_graph.install_order()
 
+        package_count = sum([sum(len(install_reference.packages.values())
```

Comment:
This will be the count of all packages to install, not packages to build.
I thought you wanted to put the count of the packages to build from sources only, but this is good too. I'd probably add the count information in the ``download_bulk`` output

Reply:
> I'd probably add the count information in the download_bulk output

I've added the total count to that message, as doing the N of M report seems more complicated given that _download_package can be run in parallel